### DISCUSSION TITLE
Google Drive Issues after updates

### DISCUSSION THREAD

**Discussion by jotazetas** [iterate-ch/cyberduck#16381] [Q&A]
Hi. Since updating my Mac to Sequoia, I've been experiencing issues with Cyberduck and my Google Drive downloads (which I had no problems with before). Cyberduck is updated to its latest version. The issues I've been having are:

-Very low connection speeds (almost ten times slower than they used to be)

-Download interruptions, with various error messages, such as:
                                  
                                   -Premature end of Content-Length delimited message body (expected: 1,376,028,942; received: 
                                   554,531,951).
                                  The connection attempt was rejected. The server may be down, or your network may not be properly 
                                   configured.
         
                                   or

                                   -Connection reset. The connection attempt was rejected. The server may be down, or your network may not 
                                   be properly configured."

I have a very stable 700mbps connection. I never experienced any issues until the updates, and I was really satisfied with Cyberduck before.

**Comment by dkocher** [comment]
Can you try if disabling the Firewall in _System Settings… → Network → Firewall_ makes any difference.
